src/traversable/defaultOptions.ts

```diff
@@ -0,0 +1,57 @@
+import { XMLNode } from '../XMLNode';
+
+const utf8Decoder = new TextDecoder();
+
+export const decoder = {
+  decode: (array: Uint8Array) => {
+    return utf8Decoder.decode(array);
+  },
+};
+export interface OptionsType {
+  trimValues?: boolean;
+  attributeNamePrefix?: string;
+  attributesNodeName?: boolean | string;
```

Comment:
I would only allow string even if we need to change some testcases and it would be a breaking change.

Reply:
fixed